docker/pypi/wmagent/bin/manage-common.sh

```diff
@@ -219,6 +219,31 @@ _sql_db_isclean(){
     esac
 }
 
+_couch_db_isclean(){
+  # assume that we can connect to local CouchDB
+  # assume that env contains local couch user and password.
+  echo "$FUNCNAME: Checking if local CouchDB is empty (curl to wmagent_summary db name)"
+  agent_summary=$(curl -s -u "$COUCH_USER:$COUCH_PASS" http://$COUCH_HOST:$COUCH_PORT/wmagent_summary)
+
+  if ! type jq > /dev/null ; then
```

Comment:
you could write: 
```
type jq > /dev/null || { echo "$FUNCNAME: ..." ; return $(false) ;}

```
and if you wanted to suppress also the stdErr the line should have been actually:
```
type jq > /dev/null 2>&1 || { echo "$FUNCNAME: ..." ; return $(false) ;}
```

just a suggestion, though, no need to follow on this one, but the way how it is right now the `stdError` for sure won't be redirected to `/dev/null`